MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameEndToEndTests.swift

```diff
@@ -209,7 +219,7 @@ final class MSALNativeAuthSignUpUsernameEndToEndTests: MSALNativeAuthEndToEndBas
         signUpAttributesRequiredDelegate.expectation = submitAttributesExp2
 
         signUpAttributesRequiredDelegate.attributesRequiredState?.submitAttributes(
-            attributes: attributes,
+            attributes: ["country": "Ireland"],
```

Comment:
This should take advantage of the MSAL/test/integration/native_auth/common/AttributesStub.swift in[ this PR](https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2261/files) so it should be merged after

Reply:
Done.